README.md

```diff
@@ -2,12 +2,22 @@
 
 Here you will find a great daily OpenPaas tip that will teach you everything about this very big project.
 
-### How to contribute here?
+## How to contribute here?
 Do not hesitate to send a beautiful PR!
 Follow those [instructions](https://github.com/linagora/openpaas-tip-of-the-day/blob/gh-pages/CONTRIBUTING.md).
 
 Then your tip will directly be added and displayed with your name, in the [official blog](http://linagora.github.io/openpaas-tip-of-the-day/).
 
+### Running locally
+
+You can validate that your tip is as funky as required by installing [Jekyll](https://jekyllrb.com/) to generate the tips website, or run it using Docker from:
```

Comment:
wow nice tip ! But instead of browsing to https://jekyllrb.com what about browsing to http://linagora.github.io/openpaas-tip-of-the-day/about/ instead ? I'm describing here how to setup jekyll without docker.